src/webnn_native/ops/Split.h

```diff
@@ -39,9 +39,60 @@ namespace webnn_native { namespace op {
             return graph->AddSplit(this);
         }
 
+        MaybeError CalculateShape() override {
+            auto inputShape =
+                mInputs[0]->Shape().empty() ? std::vector<int32_t>{1} : mInputs[0]->Shape();
+            auto outputShape = inputShape;
+            size_t outputSize;
+            auto axis = mAxis;
+            if (axis < 0) {
+                axis += inputShape.size();
+            }
+
+            if (mSplits.size() == 1) {
+                outputSize = mSplits[0];
+                outputShape[axis] /= outputSize;
```

Comment:
Does it check whether evenly divide?

> The number must evenly divide the dimension size of input along options.axis.

Reply:
I have checked at [L68](https://github.com/webmachinelearning/webnn-native/pull/165/files/ec9573fe6a9bc5fdb48564a1b8dfc6a2a8228997#diff-f6c5b55bda0e94cf02e751979c4a6d16a0d1fcd023e7d7f12fe2431edbee032dR68). @huningxin

Reply:
looks good. thanks!